packages/sui-studio/src/runtime-mocha/index.js

```diff
@@ -15,12 +15,18 @@ const importAll = request => request.keys().forEach(request)
 const originalKarmaLoader = window.__karma__.loaded
 window.__karma__.loaded = () => {}
 
-// get all tests files available using a regex
-const testsFiles = require.context(
-  `${__BASE_DIR__}/test/`,
-  true,
-  /\.\/(\w+)\/(\w+)\/index.(js|jsx)$/
-)
+// default testFiles in case /test folder does not exist
+let testsFiles = () => {}
+testsFiles.keys = () => []
+
+try {
```

Comment:
why you opt to implement a try/catch instead of checking if the `${__BASE_DIR__}/test/` exists?
I think the second option increases the readability of the code

Reply:
While this seems to be running on node, this is browser code but bundled by Webpack.

`require.context` is a Webpack internal method that allows us to extract files from files by using regex queries statically. We can't know dynamically if there's a `test` folder present using Node.

So, the best approach is to allow Webpack to try to use require.context and, if not, just avoid the Error.

We could add a Babel plugin to do the check but it would complicate the code instead.